src/main/java/uk/gov/hmcts/probate/controller/DocumentController.java

```diff
@@ -198,6 +198,9 @@ public ResponseEntity<CallbackResponse> generateGrant(
         documents.add(digitalGrantDocument);
         documents.add(coverSheet);
         Optional<UserInfo> caseworkerInfo = userInfoService.getCaseworkerInfo();
+        if (caseData.getOutsideUKGrantCopies() != null && caseData.getOutsideUKGrantCopies() > 0) {
+            documents.add(notificationService.sendSealedAndCertifiedEmail(caseDetails));
+        }
```

Comment:
maybe this should be merged to the below if-block?
Otherwise, it will still send an email even if the request fails validateEmailRequest and a callback error is returned